db/migrations/01_create_user_table.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  email VARCHAR(40),
+  password VARCHAR(500),
+  name VARCHAR(50),
+  username VARCHAR(20))
+
+CREATE TABLE peeps (
+  id SERIAL PRIMARY KEY,
+  content VARCHAR(140),
+  date DATE,
+  time TIME,
+  user_id INT,
+  FOREIGN KEY (user_id) REFERENCES users (id)
+);
```

Comment:
Good use of clear instructions for another developer to easily set up the project